content/BindingMethods/BindingMethods.facts.md

```diff
@@ -27,7 +27,7 @@ $pstmt->execute();
 
 - `$data['first_name']` and `$data['birth_date']` represent the values to bind to the positional placeholders `?` in the SQL statement.
 
-- `PDO::PARAM_STR` represents a predefined constant `SQL CHAR`, `VARCHAR`, or other string data type for the parameter.
+- `PDO::PARAM_STR` represents a predefined constant `SQL CHAR`, `VARCHAR`, or other string data type for the parameter. A few other types of predefined constants are `PDO::PARAM_BOOL` for a boolean data type, `PDO::PARAM_NULL` for SQL null data type, and `PDO::PARAM_INT` for SQL integer data type.
```

Comment:
How about using bullets for the rest of the predefined constants?
Maybe we can have like this:

__A few other types of predefined constants are:__
 - `PDO::PARAM_BOOL` for a boolean data type
 - `PDO::PARAM_NULL` for SQL null data type
 - and so on ...